source/renderer/app/stores/NodeUpdateStore.js

```diff
@@ -64,7 +64,7 @@ export default class NodeUpdateStore extends Store {
     }
   };
 
-  @action _activateAutomaticUpdate = async nextUpdateVersion => {
+  @action _activateAutomaticUpdate = async (nextUpdateVersion: ?number) => {
```

Comment:
@tomothespian Will check this once I return back.

Reply:
@tomothespian Again, my approach is correct, what you pointed out is a `observable` value which is string.
But the parameter I used should be number.
That's fetched from here.
https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/api/nodes/types.js#L4
Please be careful when leaving comments after looking into definition of objects rather than same name variable in the file.

Reply:
@yakovkaravelov please update this observable's type and set it to number. That will match the Api response type declaration then.

Reply:
@tomothespian @nikolaglumac These two variables are actually different.
Input parameter is from `result.version` of `nextUpdate` api request which is number.
But observable parameter is determined by `availableAppVersion` observable which is generated by `latestAppVersion` of `getLatestAppVersionRequest` api result.